scripts/sdk-coin-generator/template/.blank.ts

```diff
@@ -0,0 +1,103 @@
+// Import coin specific libraries
+
+import {
+  BaseCoin,
+  BitGoBase,
+  checkKrsProvider,
+  getBip32Keys,
+  InitiateRecoveryOptions as BaseInitiateRecoveryOptions,
+  InvalidAddressError,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  promiseProps,
+  SignTransactionOptions as BaseSignTransactionOptions,
+  TransactionExplanation,
+  TransactionPrebuild,
+  UnexpectedAddressError,
+  VerifyAddressOptions as BaseVerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '@bitgo/sdk-core';
+
+// Create and export functions that interact with BitGo services
+
+// Example interfaces and classes below:
+
+interface Address {}
+
+interface FeeInfo {}
+
+interface SignTransactionOptions extends BaseSignTransactionOptions {}
+
+interface ExplainTransactionOptions {}
+
+interface VerifyAddressOptions extends BaseVerifyAddressOptions {}
+
+interface RecoveryInfo extends TransactionExplanation {}
+
+export interface InitiateRecoveryOptions extends BaseInitiateRecoveryOptions {}
+
+export interface RecoveryOptions {}
+
+interface HalfSignedTransaction {}
+
+interface SupplementGenerateWalletOptions {}
+
+export class <%= constructor %> extends BaseCoin {
+  protected constructor(bitgo: BitGoBase) {
+    super(bitgo);
+  }
+
+  static createInstance(bitgo: BitGoBase): BaseCoin {
+    return new <%= constructor %>(bitgo);
+  }
+
+  /**
+   * Factor between the coin's base unit and its smallest subdivison
+   */
+  public getBaseFactor(): number {
+    // return 1e6;
+  }
+
+  public getChain(): string {
+    return '<%= symbol %>';
+  }
+
+  public getFamily(): string {
+    return '<%= symbol %>';
+  }
+
+  public getFullName(): string {
+    return '<%= coin %>';
+  }
+
+  public getAddressDetails(address: string): Address {}
```

Comment:
each of these empty functions should return 
```
throw new NotImplementedError('<name of function> not implemented');
```

so this compiles successfully after generating